x-pack/solutions/observability/plugins/apm/public/embeddable/trace_waterfall/react_embeddable_factory.tsx

```diff
@@ -35,6 +36,8 @@ export interface ApmTraceWaterfallEmbeddableEntryProps extends BaseProps, Serial
   serviceName: string;
   entryTransactionId: string;
   displayLimit?: number;
+  scrollElement?: Element;
+  onNodeClick?: (item: IWaterfallSpanOrTransaction, flyoutDetailTab: string) => void;
```

Comment:
Should we change this to only pass the item id instead? 

```suggestion
  onNodeClick?: (id: string) => void;
```

Reply:
https://github.com/elastic/kibana/pull/222214/commits/c065b84b8cdd4ca4fee25421a1b8304d90f7f4aa